crates/uv-platform-tags/src/platform.rs

```diff
@@ -230,23 +216,22 @@ impl Arch {
         }
     }
 
-    /// Returns an iterator over all supported architectures.
-    pub fn iter() -> impl Iterator<Item = Self> {
-        [
-            Self::Aarch64,
-            Self::Armv5TEL,
-            Self::Armv6L,
-            Self::Armv7L,
-            Self::Powerpc64Le,
-            Self::Powerpc64,
-            Self::Powerpc,
-            Self::X86,
-            Self::X86_64,
-            Self::S390X,
-            Self::LoongArch64,
-            Self::Riscv64,
-        ]
-        .iter()
-        .copied()
+    /// Represents the hardware platform.
+    ///
+    /// This is the same as the native platform's `uname -m` output.
+    ///
+    /// Based on: <https://github.com/PyO3/maturin/blob/8ab42219247277fee513eac753a3e90e76cd46b9/src/target/mod.rs#L131>
+    pub fn machine(&self) -> &'static str {
+        match self {
+            Arch::Aarch64 => "arm64",
+            Arch::Armv5TEL | Arch::Armv6L | Arch::Armv7L => "arm",
+            Arch::Powerpc | Arch::Powerpc64Le | Arch::Powerpc64 => "powerpc",
```

Comment:
@konstin -- We can just wait until we get user feedback on it, but I'm surprised by this, I would've expected `powerpc64`.